amq/amq62-persistent-ssl.json

```diff
@@ -551,7 +547,7 @@
             "metadata": {
                 "name": "${APPLICATION_NAME}-drainer",
                 "labels": {
-                    "application": "${APPLICATION_NAME}"
+                    "application": "${APPLICATION_NAME}-drainer"
```

Comment:
@spolti, we need to change this back to just ``${APPLICATION_NAME}``.  The ``application`` label is intended to be used to identify all resources created by this template so they can be easily accessed via ``-l application=xyz``.